src/output.rs

```diff
@@ -89,7 +89,7 @@ impl OutputType {
                 }
 
                 if wrapping_mode == WrappingMode::NoWrapping(true) {
-                    p.arg("-S"); // Short version of --chop-long-lines for compatibility
+                    p.arg("-Sc"); // Short version of --chop-long-lines --clear-screen for compatibility
```

Comment:
I wonder if it's worth trying to add an integration test for this... I guess probably more hassle than it's worth :) 
can we be confident that `-c` flag is standard across most `less` versions/implementations in the wild?

Reply:
> I wonder if it's worth trying to add an integration test for this... I guess probably more hassle than it's worth :)

Well, in any case, that's not something I could help with, I only made this edit because you showed me where to make it :sweat_smile: 

> can we be confident that `-c` flag is standard across most `less` versions/implementations in the wild?

Definitely 🤣

[`v160   7/29/90  Clear screen before initial output w/ -c.`](https://github.com/gwsw/less/blob/25d08b356e98b34ee2c1cc63d7170e0dea87290d/version.c#L266)